docker-app/worker_wrapper/wrapper.py

```diff
@@ -264,15 +267,30 @@ def _run_docker(
             detach=True,
```

Comment:
@suricactus  Did you have bad experience with `auto_remove=True` , as you commented it?

Reply:
If I remember correctly, configuring the `auto_remove=True` will make it impossible to get the logs if the container is timed out.